projects/ccf-eui/src/app/app.module.ts

```diff
@@ -16,6 +16,11 @@ import { DualSliderModule } from './shared/components/dual-slider/dual-slider.mo
 import { SpinnerOverlayModule } from './shared/components/spinner-overlay/spinner-overlay.module';
 import { ViewerModule } from './shared/components/viewer/viewer.module';
 
+import { TrackingPopupModule, INITIAL_TELEMETRY_SETTING } from 'ccf-shared';
+import { NgxGoogleAnalyticsModule } from 'ngx-google-analytics';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { environment } from '../environments/environment.staging';
```

Comment:
Don't import specific environments like that. This should always be:
`import { environment } from '../environments/environment';`
When building, the angular cli will inject the right environment.